packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.controller.js

```diff
@@ -1,16 +1,23 @@
-import { NSX_RESOURCES } from './dedicatedCloud-datacenter-manage-nsx.constants';
+import {
+  NSX_RESOURCES,
+  TRACKING,
+} from './dedicatedCloud-datacenter-manage-nsx.constants';
 import { EDGES_SIZES } from '../../../datacenters/datacenter.constants';
 
 export default class {
   /* @ngInject */
-  constructor(DedicatedCloud, $translate, coreURLBuilder) {
+  constructor(DedicatedCloud, $translate, coreURLBuilder, atInternet) {
     this.DedicatedCloud = DedicatedCloud;
     this.$translate = $translate;
     this.coreURLBuilder = coreURLBuilder;
     this.NSX_RESOURCES = NSX_RESOURCES;
+    this.TRACKING = TRACKING;
+    this.atInternet = atInternet;
   }
 
   $onInit() {
+    this.atInternet.trackPage(this.TRACKING.NSX);
```

Comment:
**nitpick:** We should disable the default tracking applied at the state change by adding 
```
atInternet: {
    ignore: true
}
```